src/erlang_ls_utils.erl

```diff
@@ -12,16 +13,25 @@
 %%
 %% Look for a given module in the DB and return the respective
 %% `URI`. If the module is not in the DB, try to index it.
+
 -spec find_module(atom()) -> {ok, uri()} | {error, any()}.
 find_module(M) ->
-  case erlang_ls_db:find(completion_index, M) of
+  find_module(M, erl).
+
+-spec find_module(atom(), erl | hrl) -> {ok, uri()} | {error, any()}.
+find_module(M, Extension) ->
```

Comment:
I modified this function so it could also be used for finding and indexing `hrl` files.